### PR TITLE
Travis UI tests flakyness

### PR DESCRIPTION
**Pull request by CristianGM**

### CODE REVIEW

**Comment by ferranpons** [comment]
@CristianGM What could we do with this? Worth keeping it or do we close it?

File: .travis.yml [hunk 1]
```diff
@@ -39,16 +39,17 @@ android:
   - sys-img-armeabi-v7a-google_apis-21
 
 # Emulator Management: Create, Start and Wait
-#before_script:
-#  - android list targets
-#  - echo no | android create avd --force -n test -t android-21 --abi armeabi-v7a --tag google_apis
-#  - emulator -avd test -no-skin -no-audio -no-window &
+before_script:
+  - android list targets
```
Comment (alorma) [inline .travis.yml]: It's like update?

File: .travis.yml [hunk 2]
```diff
@@ -39,16 +39,17 @@ android:
...
+  - echo no | android create avd --force -n test -t android-21 --abi google_apis/armeabi-v7a
+  - emulator -avd test -no-window -gpu off -no-boot-anim &
+  - android-wait-for-emulator
+  - travis_wait 20 android-wait-for-emulator
```
Comment (alorma) [inline .travis.yml]: why two `android-wait-for-emulator`?

File: .travis.yml [hunk 3]
```diff
@@ -39,16 +39,17 @@ android:
...
+  - adb shell input keyevent 82 &
```
Comment (alorma) [inline .travis.yml]: a comment saying what `82` means will be good, so w. don't change it by accident

File: leku/src/androidTest/java/com/schibsted/leku/LocationPickerActivityShould.java [hunk 4]
```diff
@@ -71,6 +97,8 @@ public void showSuggestedLocationListWhenATextSearchIsPerformed() throws Excepti
         .check(matches(hasImeAction(EditorInfo.IME_ACTION_SEARCH)))
         .perform(pressImeActionButton());
     wait300millis();
+    wait300millis();
+    wait300millis();
```
Comment (alorma) [inline leku/src/androidTest/java/com/schibsted/leku/LocationPickerActivityShould.java]: `wait600milis()`? :p

also; wait uses Barista.sleep?

Comment (ferranpons) [inline leku/src/androidTest/java/com/schibsted/leku/LocationPickerActivityShould.java]: is not using barista right now

File: .travis.yml (hunk 2 above)
Comment (ferranpons) [inline .travis.yml]: this was a test. removed

File: .travis.yml (hunk 1 above)
Comment (ferranpons) [inline .travis.yml]: no, only lists the emulators available. Not needed anymore

**Comment by ferranpons** [comment]
In reality Tests aren't flaky, the problem now is that there are several tests that don't pass anymore. So I need to solve what is broken. Now this is more related with the issue #75.

That's why I changed this PR to WIP. I'm going to implement the fallback for the location that is not being displayed for low performant devices.

We need to be careful with UI tests because the task that execute them (connectedDebugAndroidTest) is really slow. It doesn't show any log for almost 9 minutes. That's why could crash the build. Travis has a timeout of 10 minutes. 😩

**Pull request merged**